src/pages/infrastructure.js

```diff
@@ -94,6 +94,17 @@ const emergingProjects = [
       {label: 'Slack channel', url: 'https://cilium.slack.com/archives/C0210QTK2MV'},    
     ],
   },
+  {
+    name: 'uBPF',
+    title: 'uBPF - Userspace eBPF Runtime',
```

Comment:
Nit: Title for other runtimes is just `eBPF Runtime`, more like a category than a real title, maybe worth keeping the trend. Otherwise, s/`Userspace`/`User space`/, for consistency with the Linux description?

Reply:
In this case I'd probably keep the subtitle as is (modulo your 2nd suggestion) to make it clear it's a runtime for user space to avoid potential confusion.

Reply:
Just `User Space eBPF Runtime` in that case? `uBPF` will be written right above already. We could fix hBPF too.

Reply:
Yes, I think just User Space eBPF Runtime. hBPF should be fixed too. I'll create a PR